rotkehlchen/db/schema.py

```diff
@@ -676,6 +676,16 @@
 );
 """
 
+DB_CREATE_MAPPED_ACCOUNTING_RULES = """
+CREATE TABLE IF NOT EXISTS linked_rules_properties(
+    identifier INTEGER PRIMARY KEY NOT NULL,
+    accounting_rule INTEGER REFERENCES accounting_rules(identifier),
+    property_name TEXT NOT NULL,
+    setting_name TEXT NOT NULL references settings(name),
+    active BOOLEAN
```

Comment:
What's the difference between this and `INTEGER NOT NULL CHECK (status IN (0, 1))`? I think there must have been a reason we are goign with the explicit `INTEGER NOT NULL CHECK (status IN (0, 1))` since it actively fails if you try to pass something like `2` there. Will the `BOOLEAN` also do that? I see nothing in the docs.

Reply:
yes we need to use the check. I recall this discussion before and I used boolean because I was thinking in sql and not sqlite. 

```
SQLite does not have a separate Boolean storage class. Instead, Boolean values are stored as integers 0 (false) and 1 (true).

SQLite recognizes the keywords "TRUE" and "FALSE", as of version 3.23.0 (2018-04-02) but those keywords are really just alternative spellings for the integer literals 1 and 0 respectively.
```

https://www.sqlite.org/datatype3.html